astrocut/tests/test_cutout_processing.py

```diff
@@ -210,7 +212,8 @@ def test_configure_bintable_header(tmpdir):
     # TODO: add test where there are more than one cutout headers
 
 
-def test_center_on_path(tmpdir):
+@pytest.mark.parametrize("in_target, out_file", [(None, "path_"), ("C/ Targetname", "C-_Targetname")])
```

Comment:
This will do Test 1 twice. We should either pull out test 1 into a separate test function, or possibly configure it here as a separate test in the parametrize args.

Reply:
I've moved Test 1 to its own function called `test_center_on_path_input_tpf` so it can run independently.